examples/notebooks/get-started/try-apache-beam-yaml.ipynb

```diff
@@ -0,0 +1,556 @@
+   ],
+   "execution_count": null,
+   "outputs": []
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Inspect the data\n",
+    "We'll be working with 2 datasets. We'll use `kinglear.txt` for the first example - word count, and `SMSSpamCollection.csv` for the second and third.\n",
+    "Let's first take a loot at the `kinglear.txt` dataset."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "run('head data/kinglear.txt')"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "This is just a `txt` file - it contains lines of text.\n",
+    "Let's take a look at the other dataset."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "run('head data/SMSSpamCollection.csv')\n",
+    "run('wc -l data/SMSSpamCollection.csv')"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "This dataset is a `csv` file with 5,574 rows and 2 columns recording the following attributes separated by a tab sign:\n",
+    "1. `Column 1`: The label (either `ham` or `spam`)\n",
+    "2. `Column 2`: The SMS as raw text (type `string`)"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Example 1: word count\n",
+    "In this popular introductory exercise, we will build a pipeline that reads lines of text from the input dataset `kinglear.txt` and counts the number of times each word appears in the text.\n",
+    "To start, we'll create a `.yaml` file specifying our pipeline."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "pipeline = '''\n",
+    "pipeline:\n",
+    "  # Read input data. Each line from the txt file is a String.\n",
+    "  - type: ReadFromText\n",
+    "    name: InputText\n",
+    "    file_pattern: data/kinglear.txt\n",
+    "\n",
+    "  # Using a regex, we'll split the content of the message (one long string) into words (list of strings).\n",
+    "  # The 'fn' parameter accepts functions written in Python\n",
+    "  - type: PyFlatMap\n",
+    "    name: FindWords\n",
+    "    input: InputText\n",
+    "    fn: |\n",
+    "      import re\n",
+    "      lambda line: re.findall(r\"[a-zA-Z]+\", line)\n",
+    "\n",
+    "  # Transforming each word to lower case and combining it with a '1'. Result of this step are pairs (word: 1).\n",
+    "  - type: PyMap\n",
+    "    name: PairWordsWith1\n",
+    "    input: FindWords\n",
+    "    fn: 'lambda word: (word, 1)'\n",
+    "\n",
+    "  # Using SumPerKey transform, we'll calculate the occurrence of each word.\n",
+    "  - type: SumPerKey\n",
```

Comment:
This is just a test transform, right?

Reply:
Removed, changed to CombinePerKey with a callable argument